api/plugins/ginfo/ginfo.py

```diff
@@ -0,0 +1,153 @@
+
+"""
+  Integration into the Ginfo Map via Firebase REST API
+"""
+from django.http import HttpResponse, HttpRequest
+from django.db.models import Q
+import requests
+import math
+from api.models import Character
+from ginfocharacter import GinfoCharacter
+from datetime import datetime
+from firebase_pushid import PushID
+
+
+class GinfoPlugin(object):
+    """
+      GinfoPlugin provides methods to post a position to the ginfo firebase database
+    """
+    API_URL = 'https://ginfo-34baa.firebaseio.com'
+    GAME = 'conanexiles'
+    MAP = 'conanexiles_desert'
+    GINFO_USER_UID = 'y67Efa3kddSuhb26UoUapIJakMh1'
```

Comment:
This is the UID of a User I've created specifically for ServerThrall, but its just an ordinary user. Having its UID public should be no problem. The only request possible without authentication is to post markers like ServerThrallAPI does, but you still need the access token for a group to do that. 
If you're an authenticated user, you could theoretically send group invites to this user. Other than that, theres nothing you can with the UID.

Right now we have it so that either everyone in a group can move/delete every marker, or at least admin can move/delete every marker (this is currently used only in special groups).

I was thinking about requiring the Users UID as a configuration parameter as well like the Group UID. But its nice to have a way for us to query for markers created by ServerThrall for statistics.